src/logger.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ *
+ * Handles team-based logging for developers in PROD
+ *
+ * Prevents a proliferation of console.log in client-side
+ * code.
+ *
+ * Team registration relies on LocalStorage
+ */
+
+import { storage } from './storage';
+
+type TeamColors = Record<string, Record<string, string>>;
+
+const teamColors: TeamColors = {
+	common: {
+		background: '#052962',
+		font: 'white',
+	},
+	commercial: {
+		background: 'mediumseagreen',
+		font: 'darkgreen',
+	},
+};
+
+const style = (team: string): string => {
+	const { background = 'black' } = { ...teamColors[team] };
+	const { font = 'white' } = { ...teamColors[team] };
```

Comment:
Do you need the default? You have common as the base style on line 48, and then the type for TeamColo(u)rs (😉), does not allow for optional values in the background and font.

Reply:
This would allow new team to start using the tool before they’re registered in the teams array. The default is used to style the `@guardian` part of the log. Next comes the team label, and third what the arguments are.

![console logger message](https://user-images.githubusercontent.com/76776/101355718-f8fac080-3864-11eb-925f-4f26132ab92f.png)

Reply:
Ah, ok! It's beautiful.